database_admin/update.sh

```diff
@@ -33,6 +33,3 @@ psql -c "ALTER USER evaluator WITH PASSWORD '${EVALUATOR_PASSWORD}'"
 psql -c "ALTER USER manager WITH PASSWORD '${MANAGER_PASSWORD}'"
 psql -c "ALTER USER vmaas_sync WITH PASSWORD '${VMAAS_SYNC_PASSWORD}'"
 psql -c "ALTER USER cyndi WITH PASSWORD '${CYNDI_PASSWORD}'"
-
```

Comment:
It should be done by database_admin. We can not rely on adding it directly to container, because it's not possible in prod where we use RDS database. Moreover current solution works everywhere but in testing db. Let's fix only this place.

Reply:
Note that in order to fix this in database_admin, it has to be done by superuser. Will there be any trouble with granting database_admin with superuser role?

Reply:
We need to consider other environments. I think we don't use the flag, so we should probably remove it alltogether.